src/graphql/search/filters/index.ts

```diff
@@ -0,0 +1,21 @@
+import { Filter } from '../../../generated/graphql'
+import { Context } from '../../config'
+import getCmsFilters from '../cms/filters'
+import { DCAT_ENDPOINTS } from '../datasets/config'
+import getDatasetsFilters from '../datasets/filters'
+import { combineFilters } from './utils'
+
+export default async (_: any, {}, { loaders }: Context): Promise<Filter[]> => {
+  // Get the drupal theme taxonomy from the DataLoader
+  const cmsThemeTaxonomy: { value: any } = await loaders.cms.load(
```

Comment:
no error handling?

Reply:
I assume the error propagates up and is handled there?